app/packs/src/decidim/vocdoni/admin/available_credits.js

```diff
@@ -46,6 +46,12 @@ const collectFaucetTokensListener = async () => {
   });
 }
 
+export const getAvailableCredits = async () => {
```

Comment:
I think we should refactor the showAvailableCredits function, for remove the duplication of logic. I didn't actually tried but this should be enough:  

```javascript
const showAvailableCredits = async () => {
  const creditsSpan = document.querySelector(CREDIT_SPAN_SELECTOR);
  if (!creditsSpan) {
    return;
  }

  const availableCredits = await getAvailableCredits();

  if (availableCredits === 0) {
    document.querySelector(NO_TOKENS_MESSAGE_SELECTOR).classList.remove("hide");
    document.querySelectorAll(ACTIONS_BUTTONS_SELECTOR).forEach((element) => {
      element.disabled = true;
    });
  }

  creditsSpan.innerHTML = availableCredits;
};
```

Reply:
I refactored it.